components/ModalTemplateSwitchExtensionNetwork.vue

```diff
@@ -17,7 +12,7 @@ const handleChangeNetwork = async () => {
   } catch (error: unknown) {
     console.error(error)
   } finally {
-    props.closeModal()
+    closeModal()
```

Comment:
wouldnt you need to do `await` here?

Reply:
Seem to be not necessary but wont hurt ;)